pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -579,6 +586,8 @@ spec:
               mountPath: /var/run
             - name: conf
               mountPath: /usr/local/etc/istgt
+            - name: storagepath
+              mountPath: {{ .Config.PersistentStoragePath.value }}
```

Comment:
why PersistentStoragePath *variable* is needed? it  has to be "/var/openebs" all the time, other wise istgt image needs to be regenerated with the new mount path? isn't it?

Reply:
Yes, If that variable changed then istgt image also needs to be regenerated. So one shouldn't touch that `PersistentStoragePath` variable(Made comments above that variable).
We are using in different places so made variable to make it use.

Reply:
I will hard code it